### PR TITLE
BD-3235 Add SMS Total Clicks info

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

Add Total Clicks definitions and calculations where missing

### Related PRs, issues, or features (optional)

[#BD-3235](https://jira.braze.com/browse/BD-3235)

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/sms/campaign/link_shortening.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ When link shortening is enabled, the SMS and MMS performance table include a col
 
 ![][4]
 
-The Historical Performance and SMS Overview chart also includes an option for  **Total Clicks** and shows a daily time series of click events.
+The Historical Performance and SMS Overview chart also includes an option for  **Total Clicks** and shows a daily time series of click events. Clicks are incremented on redirect (such as when a user visits the same link multiple times), and may be incremented more than once per user.
```
Comment (rachel-feinberg) [inline _docs/_user_guide/message_building_by_channel/sms/campaign/link_shortening.md:68]:
```suggestion
The Historical Performance and SMS Overview chart also includes an option for  **Total Clicks** and shows a daily time series of click events.
```

File: _includes/campaign_analytics.md [hunk 2]
```diff
@@ -299,6 +299,7 @@ Here is a breakdown of some key metrics you may see while reviewing your message
 | Rejections | The SMS has been rejected by the carrier. This can happen for several reasons, including carrier content filtering, availability of the destination device or the phone number is no longer in service. As a Braze customer, rejections are charged toward your SMS allotment. |
 | Opt-Out | A user replied to your message with an [opt-out keyword]({{site.baseurl}}/user_guide/message_building_by_channel/sms/keywords/keyword_handling/#default-opt-in-opt-out-keywords) and was unsubscribed from your SMS program. A user reply is measured anytime a user sends an inbound message within four hours of receiving your message. |
 | Help | A user replied to your message with a [HELP keyword]({{site.baseurl}}/user_guide/message_building_by_channel/sms/keywords/keyword_handling/#default-opt-in-opt-out-keywords) and was dispatched a HELP auto-response. A user reply is measured anytime a user sends an inbound message within four hours of receiving your message. |
+| Total Clicks | The total number (and percentage) of users who clicked within the delivered email, card, or message. |
```
Comment (bre-fitzgerald) [inline _includes/campaign_analytics.md:302]: Given that this table is specific to SMS, we should be specific in the description too!
```suggestion
| Total Clicks | The total number (and percentage) of users who clicked within the delivered message. |
```

**Review by bre-fitzgerald — APPROVED** [review]
Lgtm! Just left one small suggestion. Feel free to merge at your leisure 🫡

File: _includes/campaign_analytics.md (hunk 2 above)
Comment (rachel-feinberg) [reply _includes/campaign_analytics.md:302]: Thank you for catching!

**Pull request merged**